packages/pools/src/constants/pools/56.ts

```diff
@@ -13,6 +13,16 @@ export const livePools: SerializedPool[] = [
     tokenPerBlock: '10',
     isFinished: false,
   },
+  {
+    sousId: 366,
+    stakingToken: bscTokens.cake,
+    earningToken: bscTokens.ckp,
+    contractAddress: '0x87f0210c658c81e854e6022315cD68804944acaE',
+    poolCategory: PoolCategory.CORE,
+    tokenPerBlock: '0.001929',
+    isFinished: false,
```

Comment:
remove this

Reply:
@ChefBingbong due to the timeline I'll merge anyway. Please take a look at this comment later